src/resources/js/tickets.js

```diff
@@ -64,6 +65,224 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 		}
 	};
 
+	function format_date(date) {
```

Comment:
Can we replace these two methods (time management) by using moment.js? It's been used already. (I think)

Reply:
Oh, gawd yes - `moment.js` would be wonderful 😉